backend/src/main/resources/application.yaml

```diff
@@ -145,5 +145,7 @@ datahub:
   jwt-scope: "simple_report.*.report"
   signing-key: ${DATAHUB_SIGNING_KEY:super-secret-signing-key}
   csv-upload-api-client: "simple_report.csvuploader"
+  csv-upload-api-fhir-client: "simple_report.fullelr"
+  fhir-enabled: true
```

Comment:
I thought we wanted this to be off for now!

Reply:
The last conversation I remember having I thought we landed on "We can just start sending things because they won't go anywhere"

Reply:
Makes sense, did we double check with the RS team in that case? just so they know